### PR TITLE
libbeat/version: fix version syntax

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

The version was incorrectly written by automation as a non-semver string. This fixes that.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by ycombinator — APPROVED** [review]
LGTM.

File: testing/environments/latest.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:8.9.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.8.1
```
Comment (cmacknz) [inline testing/environments/latest.yml:6]: This should be 8.9.0 for the latest release I think, no idea why it ended up as 8.9.2 originally.

Comment (efd6) [reply testing/environments/latest.yml:6]: OK. I'll do a manual change of this.

**Review by cmacknz — APPROVED** [review]
Thanks! Will look into the release scripts to fix this for next time.

**Comment by efd6** [comment]
I think it would be really helpful if the automation noted its actions in the commit message something like this https://pkg.go.dev/rsc.io/rf/git-generate.

Making the changes go through CI, which doesn't appear to happen, would also be worthwhile since it would catch this kind of breakage.

File: testing/environments/latest.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:8.9.2
```
Comment (ruflin) [inline testing/environments/latest.yml:6]: How is this change related?

Comment (ruflin) [reply testing/environments/latest.yml:6]: Found the notes on Slack, this is to get the tests green.

**Review by ruflin — APPROVED** [review]

File: testing/environments/latest.yml (hunk 2 above)
Comment (efd6) [reply testing/environments/latest.yml:6]: Yes, this is to revert fix an incorrect automation change. It is still not passing though, so there are deeper issues.

**Comment by pierrehilbert** [comment]
Resolved in https://github.com/elastic/beats/pull/36354
